grunt/tasks/jest.js

```diff
@@ -6,25 +6,87 @@
 
 'use strict';
 
+var assign = require('object-assign');
+var async = require('async');
+var fs = require('fs');
+var glob = require('glob');
 var grunt = require('grunt');
 var path = require('path');
 
+var rootPath = path.resolve('.');
+var buildPath = path.join(rootPath, 'build');
+var tempConfigPath = path.join(buildPath, 'jest-config.json');
+
+var config = require(path.join(rootPath, 'package.json')).jest;
+
+function getCollectCoverageOnlyFrom(callback) {
+  var coverageFrom = config.collectCoverageOnlyFrom;
+  var patterns = Object.keys((config.collectCoverage && coverageFrom) || {});
+  var result = {};
+
```

Comment:
This globs each pattern of `config.collectCoverageOnlyFrom` with any glob options applied and adds the matches to the new `collectCoverageOnlyFrom` result.